data/free worlds intro.txt

```diff
@@ -1740,6 +1820,7 @@ mission "FW Bounty 3"
 	on complete
 		"assisted free worlds" ++
 		payment 300000
+		log `Killed a number of pirates for the Free Worlds, including an Osprey with some strange alien weapons. The officer I spoke to said he would tell the Free Worlds Council about me.`
```

Comment:
It's not explicitly stated that the weapons are of an alien nature. It's just conjecture. (I mean, they are, obviously, but the Free Worlds personnel have no way of knowing for sure.)

I think we can get a nicer ending than just "telling the Council about me".